api/pom.xml

```diff
@@ -278,6 +278,21 @@
 			<groupId>com.sun.xml.bind</groupId>
 			<artifactId>jaxb-impl</artifactId>
 	    </dependency>
+	   <dependency>
+		   <groupId>org.junit.jupiter</groupId>
+		   <artifactId>junit-jupiter-api</artifactId>
+		   <version>${junitVersion}</version>
```

Comment:
I believe we should add `<scope>test</scope>` to every junit dependency so these dependencies are not packaged into the api jar

Reply:
see for example https://github.com/junit-team/junit5-samples/blob/main/junit5-jupiter-starter-maven/pom.xml#L22

Reply:
@teleivo updated